CONTRIBUTING.md

```diff
@@ -90,7 +90,7 @@ The DCO requires a sign-off message in the following format appear on each commi
 Signed-off-by: Julia Child <juliachild@chef.io>
 ```
 
-The DCO text can either be manually added to your commit body, or you can add either **-s** or **--signoff** to your usual git commit commands. If you forget to add the sign-off you can also amend a previous commit with the sign-off by running **git commit --amend -s**. If you've pushed your changes to GitHub already you'll need to force push your branch after this with **git push -f**.
+The DCO text can either be manually added to your commit body, or you can add either **-s** or **--signoff** to your usual git commit commands. If you are using the GitHub UI to make a change you can add the sign-off message directory to the pull request description. If you forget to add the sign-off you can also amend a previous commit with the sign-off by running **git commit --amend -s**. If you've pushed your changes to GitHub already you'll need to force push your branch after this with **git push -f**.
 
 ### Chef Obvious Fix Policy
```

Comment:
In this following section, the "obvious fix" policy discusses an exemption to the DCO requirement for small things... but as far as I know that is not true for the `automate` repository: if you do not have a DCO the PR will fail its requisite checks.

Reply:
Many things do not require a DCO signoff. https://github.com/chef/chef/blob/master/CONTRIBUTING.md#chef-obvious-fix-policy

Reply:
This was the policy that legal drafted. Expeditor is simply enforcing sign-offs on a PR. What's written here is what we should be enforcing.